domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -2545,7 +2545,7 @@ public ResultsHelper(Class<T> clazz) {
     return (int) list.stream()
         .filter(e -> !(e instanceof Facet) &&
                      !((e instanceof Map) &&
-                         ((Map) e).size() == 1 && ((Map) e).containsKey(COUNT_FIELD)))
+                      ((Map) e).size() == 1 && ((Map) e).containsKey(COUNT_FIELD)))
```

Comment:
This indentation change is misleading because the `!` in the previous line also includes this line.

Reply:
What my editor does. I'll reformat.

Reply:
Going away completely!